helm-chart/values.yaml

```diff
@@ -8,7 +8,7 @@ image:
   repository: andrewmoy/globaleaks
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
-  tag: helm
+  tag: "4.13.18"
```

Comment:
i wonder if we could create a tag "stable" to be always mapped on the lates stable tag

@rglauco @gianlucagilardi @andrewmoyaccrete : would this be helpful in your opinion?

Reply:
I am indifferent on this, will do whatever you guys decide